### DISCUSSION TITLE
12h extra condensed vertical taskbar clock

### DISCUSSION THREAD

**Discussion by Optissimum** [MarcinOrlowski/html-clock-plasmoid#61] [HTML Clock templates]
```html
<html>
<body>
<center>
<span style="font-family: Fira Sans Extra Condensed, Noto Sans Extra Condensed, Fira Sans Condensed, Liberation Sans Narrow, Fira Sans, Noto Sans; font-weight: bold; font-size: 14pt; color: #FFFFFF;">{k}</span>
<span style="font-family: Fira Sans Extra Condensed, Noto Sans Extra Condensed, Fira Sans Condensed, Liberation Sans Narrow, Fira Sans, Noto Sans; font-weight: light; font-size: 15pt; color: #FFFFFF;"> {ii}</span>
</center>
</body>
</html>
```
![image](https://user-images.githubusercontent.com/16561967/188719489-ffca3503-289d-4c27-9594-16c761e722e6.png)

** Notes **
Designed for vertical taskbars, attempting to use an Extra Condensed font and only bold/light font weights to keep horizontal space minimal. Uses 12h clock.

** Confirmation **

* [X] I confirm this is my original work.
* [X] I attached screenshot of my layout to this ticket.
* [X] I grant full and unrevocable rights to use the layout with HTML Clock widget free of charge.
* [X] I grant users of HTML Clock widget right to use my layout free of charge and freely alter my layout if they need to.
* [X] I want to be credited as Optissima.
* [X] This layout is licensed under GPL2.0

**Comment by MarcinOrlowski** [comment]
Thanks for your contribution
